docs/rules/ultiduo.md

```diff
@@ -24,10 +24,19 @@ Failure to observe weapon whitelisting will result in league penalties. For Ulti
 - **Medic**
     - Syringe Gun & Syringe Gun Reskins
     - Crusader's Crossbow
+ 
+### 1.3. Weapon Pickups
+Weapons dropped on the ground by dead players are **allowed** to be picked up by surviving players - including the medigun.
 
-### 1.3. Map Scoring
+### 1.4. Map Scoring
 In Ultiduo, all maps are King of the Hill (KOTH) maps, and follow a best of one win condition with a five minute team clock. Different KOTH maps may vary slightly on overtime timers, locked and unlocked timers (when the point is not able to be captured at the start and when it unlocks) as these are built into the map.
 
+### 1.5 Tie Breakers
+As the KOTH map standard, each map will be placed until a team has scored 3 rounds. No Tie Breaker should be needed for individual maps. 
```

Comment:
Tiebreakers should discuss Citadel Table Tiebreakers - which we haven't decided on yet. Map tiebreakers won't be needed as the config does not allow for ties.

Reply:
changed wording to allow to push - still needs work